code/obj/item/go_board.dm

```diff
@@ -0,0 +1,196 @@
+/obj/item/goboard
+	name = "go board"
+	desc = "it's a board for playing go!"
+	icon = 'icons/obj/items/items.dmi'
+	icon_state = "go-board"
+	var/list/openwindows = list()
```

Comment:
Hmm, this is just an idle thought really but I wonder if the current `updateUsrDialog` system could be made to somehow work here instead of reinventing the wheel. But it's probably more hassle than it'd be worth since you don't have a single refresh action and instead send information pecewise.

Reply:
I'm happy with how it works right now and I don't really want to break anything in how the information is relayed between players.